src/Views/Appearance.vala

```diff
@@ -76,6 +92,51 @@ public class Appearance : Gtk.Grid {
 
         update_text_size_modebutton (interface_settings);
 
+        Pantheon.AccountsService? pantheon_act = null;
+        FDO.Accounts? accounts_service = null;
```

Comment:
This variable can be reduced in scope. Remove this line and add `FDO.Accounts?` at the beginning of the line after try.